src/app/[locale]/leads/page.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import LeadsFilter from 'components/LeadsFilter/LeadsFilter';
+
+function Leads(): JSX.Element {
+  return (
+    <>
+      <div>Leads</div>
```

Comment:
https://www.notion.so/Git-rules-849f8ca8b3db4ded926d762c6fff8798 1.3.3

Reply:
Lyudmila, I fixed it, but I want you to know that this is just a sceleton for development [urposes, we already have a normal leads page in Sergii's PR. Also, this filter is already integrated in Sergii's PR. I was not going to merge it even, just wanted to get it approved and close for my task to be finished.

Reply:
Also, I didn't understand your previous comment ("this should be checked, I think as its hook useForm it should not recreating"), but I can't answer on it directly, don't know why. So, I'll answer here. Everything is working fine. And worked fine before. You want me to transfer default values back to the filter?

Reply:
Also, I didn't understand your previous comment ("this should be checked, I think as its hook useForm it should not recreating"), but I can't answer on it directly, don't know why. So, I'll answer here. Everything is working fine. And worked fine before. You want me to transfer default values back to the filter?